qpc/messages.py

```diff
@@ -116,6 +116,10 @@
 SCAN_RESTARTED = 'Scan "%s" restarted.'
 SCAN_CANCELED = 'Scan "%s" canceled.'
 
+DISABLE_OPT_PRODUCTS_HELP = 'The product inspection exclusions. '\
+    'Contains the list of products to exclude from inspection. '\
+    'Valid choices are jboss_eap, jboss_fuse, and jboss_brms.'
+
```

Comment:
@jlprevatt Would you look at the message that I added and give feedback/let me know if changes are needed? Thank you!

Reply:
For the help strings that contain a list of possible values for an option, I've tried to settle on the following standard structure for the "values" sentence:
Valid values: jboss_eap, jboss_fuse, jboss_brms.

Reply:
Awesome, I will fix it. Thanks Jamie!